tests/tests.js

```diff
@@ -147,7 +147,21 @@ var runTests = function runTests(arrayFrom) {
 		t.end();
 	});
 
-	test('no setters are called for indexes', { 'skip': !Object.defineProperty }, function (t) {
+	var defineProperty = (function () {
```

Comment:
This information is already available in the project - `require('define-properties').supportsDescriptors`